spacepy/pybats/bats.py

```diff
@@ -308,7 +285,7 @@ class Extraction(PbData):
     :func:`~spacepy.pybats.bats.Bats2d.extract`)
 
     Parameters
-    ----------
+    ==========
```

Comment:
Not going to point out the rest of these, either.

Reply:
resolved